nautobot_ssot_infoblox/utils/client.py

```diff
@@ -251,27 +279,83 @@ def get_all_ipv4address_networks(self, prefixes):
             }
         ]
         """
+
+        def get_ipaddrs(url_path: str, data: dict) -> list:
+            """Retrieve IP addresses specified in data payload.
+
+            Args:
+                url_path (str): The URL path to the API endpoint for requests.
+                data (dict): The data payload query of IP Addresses.
+
+            Returns:
+                list: List of dicts of IP Addresses for the specified prefixes or an empty list if no response.
+            """
+            response = None
+            try:
+                response = self._request(method="POST", path=url_path, data=data)
+            except HTTPError as err:
+                logger.info(err.response.text)
+            if response and len(response.json()) > 0:
+                logger.debug(response.json()[0])
```

Comment:
should we log this only when debug level is requested?

Reply:
This was done as some responses returned an empty list and blew up the import process. The logging was put there by @FragmentedPacket as he originally wrote that part of the code. IMO we should only log for debugging purposes. I don't see a reason to log it otherwise.

Reply:
ok, maybe create an issue to clean all both log topics in another PR?

Reply:
Opened #119